streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -183,19 +213,27 @@ private void encodeActiveAndStandbyTaskAssignment(final DataOutputStream out) th
     }
 
     private void encodeVersionTwo(final DataOutputStream out) throws IOException {
-        out.writeInt(2); // version
+      //  out.writeInt(2); // version
         encodeActiveAndStandbyTaskAssignment(out);
-        encodePartitionsByHost(out);
+        encodeTasksByHost(out);
```

Comment:
Changing old version encoding would break our upgrade path, ie, all existing `encodeVersionX()` methods cannot be modified.